packages/firebase_analytics/firebase_analytics/example/lib/main.dart

```diff
@@ -67,18 +67,18 @@ class _MyHomePageState extends State<MyHomePage> {
   }
 
   Future<void> _sendAnalyticsEvent() async {
+    EventParameters params = EventParameters();
```

Comment:
Maybe we could have something like EventParameters.fromMap({...}) that does all the checks in one time? It would ease migration

Reply:
Yeah, I was thinking of that. I'll add that method 👍

Reply:
I think this is something we should bring up in the meeting to be sure that items are not available for custom events as well 🤔. My own testing says no but to be 💯 sure.